web/tests/fork-workspace.spec.ts

```diff
@@ -8,11 +8,11 @@ let workspace: Workspace;
 test('fork workspace', async function ({ browser }) {
     splash = await Splash.open(browser);
     workspace = await splash.openNewWorkspace('test-example');
```

Comment:
Please change the workspace name to be specific to this test. (As long as we have to enter a name, why not use distinct names? It makes troubleshoot a bit less confusing I think.)

Reply:
Didn't you delete this test in main? #238 says it's merged into splash.spec.ts.

Reply:
Looks like I'm deleting it in #341. Which I've sent out more than a week ago! Perhaps it's time to start reviewing pull requests? 😈 (I've already merged a few where I gave up waiting. Please review these too: #336, #337, #338, #339, #345. Thanks!)

Reply:
> Looks like I'm deleting it in #341. Which I've sent out more than a week ago! Perhaps it's time to start reviewing pull requests? 😈 (I've already merged a few where I gave up waiting. Please review these too: #336, #337, #338, #339, #345. Thanks!)

This would also still be good. :point_up: 🙂